fdmi/fol_fdmi_src.c

```diff
@@ -104,6 +104,96 @@
  * rh_lsn_discarded for it's BE domain could be discarded from deduplication
  * persistence because there is nothing in the cluster that is going to send
  * FDMI FOL record about this operation.
+ *
+ * @section FDMI FOL records resend
+ *
+ * There are 2 major cases here:
+ *
+ * 1. FDMI plugin restarts, FDMI source is not. In this case FDMI source needs
+ *    to resend everything FDMI plugin hasn't confirmed consumption for. This
+ *    could be done by FDMI source dock fom by indefinitely resending FDMI
+ *    records until either FDMI plugin confirms consumption or FDMI plugin
+ *    process fails permanently.
```

Comment:
When you mentioned to me that the FDMI service is reliable.
What does it mean "FDMI plugin process fails permanently"?
What should the FDMI source do in this case?

Reply:
FDMI source could do nothing about it in this case and it would be resolved by upper layers, but FDMI source is supposed to handle this case _somehow_, because it's a valid case and any process could fail permanently. This phrase is about delivery guarantees in general - until the process is declared to be failed permanently resends should continue.

Reply:
OK